frontend/tests/controllers/ContestUsersTest.php

```diff
@@ -551,6 +551,88 @@ public function testContestPracticeWhenOriginalContestHasNotEnded() {
         }
     }
 
+    public function testNextRegisteredContestForUser() {
+        // Create 2 users
+        ['user' => $user1, 'identity' => $identity1] = \OmegaUp\Test\Factories\User::createUser(new \OmegaUp\Test\Factories\UserParams([
+            'username' => 'test_contest_user_1',
+        ]));
+        ['user' => $user2, 'identity' => $identity2] = \OmegaUp\Test\Factories\User::createUser(new \OmegaUp\Test\Factories\UserParams([
+            'username' => 'test_contest_user_2',
+        ]));
+
+        // Get the next registered contest for a user who is not registered in any contest
+        $nextRegisteredContestForUser1 = \OmegaUp\DAO\Contests::getNextRegisteredContestForUser(
+            $identity1
+        );
+        $this->assertNull($nextRegisteredContestForUser1);
+
+        $currentTime = \OmegaUp\Time::get();
+        $timePast =  new \OmegaUp\Timestamp($currentTime - 120 * 60);
+        $timeFuture1 =  new \OmegaUp\Timestamp($currentTime + 60 * 60);
+        $timeFuture2 =  new \OmegaUp\Timestamp($currentTime + 120 * 60);
+        // Get 2 active contests
```

Comment:
Hay que describir un poco más lo que hace este segmento de código:
```suggestion
        // Get 2 active contests
        // The first one started 2 hours before, and it finishes 2 hours later
        // It has a duration of 4 hours
```